mdpow/workflows/base.py

```diff
@@ -173,3 +178,34 @@ def automated_project_analysis(project_paths, ensemble_analysis, **kwargs):
 
     logger.info('all analyses completed')
     return
+
+def guess_elements(universe_atom_group):
+    # input from dihedrals module
+    # universe_atom_group = universe.atoms
+    # universe from MDPOW FEP project results
+    names = universe_atom_group.names
+    masses = universe_atom_group.masses
```

Comment:
Normally I would say "we need to add checks that we can get masses etc" but within MDPOW we now that we use TPRs so I am fine with just *documenting* it.